### PR TITLE
f-error-page@v0.1.0 - Add base component structure for f-error-page.

### PR DESCRIPTION
**Pull request by kevinrodrigues**
Base component to replace our static 404 page. I have called it `f-error-page` but totally happy to change this if you think there's a better name for it. This component will be used across our site for 404, 500 etc.

### Added
- Base generated component structure.

## UI Review Checks

- [x] README and/or UI Documentation has been [created|updated]
- [ ] Unit tests have been [created|updated]
- [ ] This code has been checked with regard to [our accessibility standards](http://fozzie.just-eat.com/documentation/general/accessibility/checklist)

## Browsers Tested

- [x] Chrome (latest)
- [ ] Internet Explorer 11
- [ ] Mobile (Please list device/browser – Ideally one iPhone model and one Android)

### List any other browsers that this PR has been tested in:

- [View the Browser Support Checklist](http://fozzie.just-eat.com/documentation/general/browser-support)

### CODE REVIEW

File: packages/components/organisms/f-error-page/package.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@justeat/f-error-page",
+  "description": "Fozzie Error Page - ",
```
Comment (dandel10n) [inline packages/components/organisms/f-error-page/package.json]: Some description should be added here

File: packages/components/organisms/f-error-page/src/components/ErrorPage.vue [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        data-test-id="errorPage">
+        {{ copy.text }}
+    </div>
+</template>
+
+<script>
+import { globalisationServices } from '@justeat/f-services';
+import tenantConfigs from '../tenants';
+/* eslint-disable */
+//import ErrorPageServiceApi from '../services/ErrorPageServiceApi';
```
Comment (zlatinivanov-je) [inline packages/components/organisms/f-error-page/src/components/ErrorPage.vue:13]: Is this needed at this stage?

**Review by dandel10n — COMMENTED** [review]
One tiny comment

**Comment by dandel10n** [comment]
Oh, I don't think you wanted to add `f-vue-icons` change to this PR

**Comment by kevinrodrigues** [comment]
> Oh, I don't think you wanted to add `f-vue-icons` change to this PR

Good spot. Not sure how that got there.

**Review by zlatinivanov-je — DISMISSED** [review]

File: packages/components/organisms/f-error-page/src/components/ErrorPage.vue (hunk 2 above)
Comment (kevinrodrigues) [reply packages/components/organisms/f-error-page/src/components/ErrorPage.vue:13]: Will probably be deleted in the next PR. Commented out for now as it was causing a eslint error.

**Comment by kevinrodrigues** [comment]
> Oh, I don't think you wanted to add `f-vue-icons` change to this PR

fixed.

File: packages/components/organisms/f-error-page/package.json (hunk 1 above)
Comment (kevinrodrigues) [reply packages/components/organisms/f-error-page/package.json]: Fixed.

**Review by simonsandersje — APPROVED** [review]

**Pull request closed without merging**